pandas/core/strings.py

```diff
@@ -2995,6 +2996,7 @@ def rindex(self, sub, start=0, end=None):
     _shared_docs['capitalize'] = dict(type='be capitalized',
                                       method='capitalize')
     _shared_docs['swapcase'] = dict(type='be swapcased', method='swapcase')
+    _shared_docs['casefold'] = dict(type='be casefolded', method='casefold')
```

Comment:
can you add the versionadded 0.25.0 here (may need to add it to the dict to be formatted)

Reply:
thanks, @jreback added!

Reply:
can u verify this renders ok in the terminal

Reply:
ahh, yeah, I printed the docstring, looks fine for me. @jreback 
```python
Convert strings in the Series/Index to be casefolded.

Equivalent to :meth:`str.casefold`.

Returns
-------
Series/Index of objects

See Also

Series.str.lower : Converts all characters to lowercase.
Series.str.upper : Converts all characters to uppercase.
Series.str.title : Converts first character of each word to uppercase and
remaining to lowercase.
Series.str.capitalize : Converts first character to uppercase and
remaining to lowercase.
Series.str.swapcase : Converts uppercase to lowercase and lowercase to
uppercase
Series.str.casefold: Removes all case distinctions in the string.
   .. versionadded:: 0.25.0
```

Reply:
Does this render well on the web? Not aware of any other instance where the version added is in the See Also section. May be worth messing around with substitution to put it in the Summary

Reply:
this rendering looks fine for me in the terminal. Since the summary of rendering only allows two parameters, and among these methods, only this `Series.str.casefold` is added in the new version. So i put this in the `See also` under `Series.str.casefold`. Otherwise, i assume the rendering might need to be changed.

Reply:
You should be able to generate one HTML file rather easily:

https://python-sprints.github.io/pandas/guide/pandas_pr.html#visual-validation-of-the-docstring

If you can double check there would be preferable as majority of users will interface to docs via HTML.

Reply:
well, do you have any idea why it's complaining `AttributeError: type object 'StringMethods' has no attribute 'casefold'`? is this quoting master? @WillAyd

Reply:
btw, i tried something like below to put version added in the summary, but when i kind of manually inserting a blank line for other methods since they don't have version added issue. I would like to hear your advice on how to improve this? @WillAyd 
```
    _shared_docs['casemethods'] = ("""
    Convert strings in the Series/Index to %(type)s.

    Equivalent to :meth:`str.%(method)s`.
    %(version)s
```

Reply:
What are you running to get the AttributeError? As far as the second comment, probably easiest to add a newline(s) into the version argument